src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/Direction.java

```diff
@@ -48,6 +53,21 @@ public Direction(
         this.tag = tag;
         this.previousDates = previousDates;
         this.clarifyingQuestions = clarifyingQuestions;
+        this.uniqueId = uniqueId;
+        this.directionType = directionType;
+    }
+
+    public Direction(
```

Comment:
where do you use this constructor? it would be good to have two new fields as mandatory ones.

Reply:
This has been refactored